python/python/benchmarks/test_v2_read_write.py

```diff
@@ -0,0 +1,403 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright The Lance Authors
+import csv
+import os
+import shutil
+from datetime import datetime
+
+import pyarrow as pa
+import pyarrow.dataset as ds
+import pyarrow.parquet as pq
+from lance.file import LanceFileReader, LanceFileWriter
+from lance.tracing import trace_to_chrome
+
+trace_to_chrome(level="trace")
```

Comment:
These trace files can clutter up the directory if not desired.  Let's not have them on by default.